src/main/java/com/spartronics4915/frc2019/subsystems/RobotStateEstimator.java

```diff
@@ -48,7 +74,14 @@ public boolean checkSystem(String subsystem)
     @Override
     public void outputTelemetry()
     {
-        // No-op
+        Pose2d odometry = mEncoderRobotState.getLatestFieldToVehicle().getValue();
+        SmartDashboard.putString("RobotState/pose",
+                odometry.getTranslation().x() +
+                        " " + odometry.getTranslation().y() +
+                        " " + odometry.getRotation().getDegrees());
+        Twist2d measuredVelocity = mEncoderRobotState.getMeasuredVelocity().getValue();
+        SmartDashboard.putNumber("RobotState/velocity", measuredVelocity.dx);
+        SmartDashboard.putNumber("RobotState/field_degrees", mEncoderRobotState.getLatestFieldToVehicle().getValue().getRotation().getDegrees());
```

Comment:
ah, here's the telemetry.. i wonder if there's a better name than field_degrees?  Perhaps RobotState/poseAngle?